### PR TITLE
James a garcia support article edits

### PR DESCRIPTION
**Pull request by Alec-Shay**
Originally from @JamesAGarcia at: https://github.com/Alec-Shay/liferay-learn/pull/24

This includes edits and improvements to the following articles:

- Application Metrics
- Configuring Your GitHub Repository
- Log Management
- Real-Time Alerts

### CODE REVIEW

**Comment by JamesAGarcia** [comment]
Regarding the Wordsmithing (1e5e2ef) commit: I just got a reply from Zeno; they usually refer to the service quick view window as the `Service Popover`. Do you think we should incorporate that into the final draft?

**Comment by Alec-Shay** [comment]
@JamesAGarcia Sorry, forgot you said you were still going to ask about that.
I had to look up "Popover" to find out that is indeed an established UI term. Going ahead and fixing that now.

File: docs/dxp-cloud/latest/en/manage-and-optimize/application-metrics.md [hunk 1]
```diff
@@ -39,18 +76,25 @@ Before using the Dynatrace advanced monitoring features, the following requireme
...
 
-Sign in with Dynatrace credentials to check log trails and create custom dashboards.
+Log in with your Dynatrace credentials to check log trails and create custom dashboards.
 
 ## Additional Information
 
-* [Advanced Monitoring: APM Tools - Dynatrace](https://help.liferay.com/hc/en-us/articles/360017896452-Advanced-Monitoring-APM-Tools-Dynatrace)
+* [Introduction to the Liferay DXP Service](https://learn.liferay.com/dxp-cloud/latest/en/using-the-liferay-dxp-service/introduction-to-the-liferay-dxp-service.html)
+* [Real-Time Alerts](https://learn.liferay.com/dxp-cloud/latest/en/manage-and-optimize/real-time-alerts.html)
+* [Quotas](https://learn.liferay.com/dxp-cloud/latest/en/manage-and-optimize/quotas.html)
```
Comment (jrhoun) [inline docs/dxp-cloud/latest/en/manage-and-optimize/application-metrics.md:99]: These can/should be relative paths to the markdown files. @JamesAGarcia @Alec-Shay

File: docs/dxp-cloud/latest/en/manage-and-optimize/real-time-alerts.md [hunk 2]
```diff
@@ -1,40 +1,49 @@
...
+
+* **Alerts Preferences**: Go to your alert preferences page.
 
 ![Real Time Alerts](./real-time-alerts/images/02.png)
+
+## Additional Information
+
+* [Application Metrics](https://learn.liferay.com/dxp-cloud/latest/en/manage-and-optimize/application-metrics.html)
+* [Quotas](https://learn.liferay.com/dxp-cloud/latest/en/manage-and-optimize/quotas.html)
+* [Team Activities](https://learn.liferay.com/dxp-cloud/latest/en/manage-and-optimize/team-activities.html)
```
Comment (jrhoun) [inline docs/dxp-cloud/latest/en/manage-and-optimize/real-time-alerts.md:49]: Same feedback as earlier - should be relative paths not absolute urls.

File: docs/dxp-cloud/latest/en/troubleshooting/log-management.md [hunk 3]
```diff
@@ -1,39 +1,47 @@
...
 
 ```shell
 lcp log -p <environment-id> -s <service-id>
 ```
+
+## Additional Information
+
+- [Shell Access](https://learn.liferay.com/dxp-cloud/latest/en/troubleshooting/shell-access.html)
+- [Disaster Recovery Overview](https://learn.liferay.com/dxp-cloud/latest/en/troubleshooting/disaster-recovery-overview.html)
+- [Configuring Cross-Region Disaster Recovery](https://learn.liferay.com/dxp-cloud/latest/en/troubleshooting/configuring-cross-region-disaster-recovery.html)
```
Comment (jrhoun) [inline docs/dxp-cloud/latest/en/troubleshooting/log-management.md:47]: Same as earlier - should be relative paths, not absolute URLs.

**Comment by jrhoun** [comment]
Merged. Thanks! https://github.com/liferay/liferay-learn/commit/f05924cdc4f4eddabd197ee079df03300d2a86e4

**Pull request closed without merging**